components/openzwave-adapter/src/lib.rs

```diff
@@ -451,7 +490,14 @@ impl OpenzwaveAdapter {
                             }
                         }
                     }
-                    ZWaveNotification::ValueRemoved(_value)         => {}
+                    ZWaveNotification::ValueRemoved(vid)            => {
+                        if let Some(getter_id) = getter_map.find_taxo_id_from_ozw(&vid) {
+                            let _ = box_manager.remove_getter(&getter_id);
+                        }
+                        if let Some(setter_id) = setter_map.find_taxo_id_from_ozw(&vid) {
+                            let _ = box_manager.remove_setter(&setter_id);
+                        }
+                    }
```

Comment:
also I think that we should remove the values from the maps getter_map and setter_map.